### PR TITLE
Improving Radio Button Behavior on the Settings Page

### PR DESCRIPTION
**Pull request by thiago-rocha**

### CODE REVIEW

File: PontoFacil/PontoFacil/Views/SettingsPage.xaml [hunk 1]
```diff
@@ -40,8 +43,10 @@
             <TimePicker Time="{x:Bind ViewModel.Profile.EntryHour, Mode=TwoWay}"/>
             <TextBlock Style="{StaticResource TextBlockStyle}" x:Uid="LunchTime"/>
             <StackPanel Orientation="Horizontal">
-                <RadioButton x:Uid="OneHour" IsChecked="{Binding LunchTimeOne, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}"/>
-                <RadioButton x:Uid="TwoHours" IsChecked="{Binding LunchTimeTwo}"/>
+                <RadioButton x:Uid="OneHour" Tag="OneHour" IsChecked="{Binding Profile.LunchTime, Mode=TwoWay,
```
Comment (soutolucas) [inline PontoFacil/PontoFacil/Views/SettingsPage.xaml]: Unnecessary code "Tag="OneHour"".

File: PontoFacil/PontoFacil/Views/SettingsPage.xaml [hunk 2]
```diff
@@ -40,8 +43,10 @@
...
+                                Converter={StaticResource LunchTimeToBoolConverter}, ConverterParameter=OneHour}" />
+                <RadioButton x:Uid="TwoHours" Tag="TwoHours" IsChecked="{Binding Profile.LunchTime, Mode=TwoWay,
```
Comment (soutolucas) [inline PontoFacil/PontoFacil/Views/SettingsPage.xaml]: Unnecessary code "Tag="TwoHour"".

File: PontoFacil/PontoFacil/ViewModels/SettingsPageViewModel.cs [hunk 3]
```diff
@@ -118,15 +102,8 @@ private void SetFirstAccess()
 
         private void SaveSettings()
```
Comment (soutolucas) [inline PontoFacil/PontoFacil/ViewModels/SettingsPageViewModel.cs]: Remove the method SaveSettings() because his have a single line.

**Review by soutolucas — CHANGES_REQUESTED** [review]
Please look the single comments.

**Comment by thiago-rocha** [comment]
@soutolucas ready to review

**Review by soutolucas — APPROVED** [review]

**Pull request merged**